plexapi/base.py

```diff
@@ -1,33 +1,48 @@
 # -*- coding: utf-8 -*-
+from __future__ import annotations
```

Comment:
for consistency, imho this must be added to any file that uses typing

Reply:
I am planning to typehint other files as well and will implement this for their respective PRs if thats fine.